### PR TITLE
EID BE 4.1.11

### PR DESCRIPTION
**Pull request by fanquake**
#### Editing an existing cask

- [x] Commit message includes cask’s name (and new version, if applicable).
- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` left no offenses.

### CODE REVIEW

File: Casks/eid-be.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask 'eid-be' do
-  version '4.1.5'
-  sha256 '1ac58ad4ef9668f0634f7648df79aed41b47789059fa5af29c0e7617b1545f2c'
+  version '4.1.11'
+  sha256 '9f70fe7c4be0676af22d7b4bdaa024e731d9b0485cf022bddffe1271ec0f7500'
 
   url "https://downloads.services.belgium.be/eid/beidbuild-#{version.delete('.')}.dmg"
```
Comment (vitorgalvao) [inline Casks/eid-be.rb]: Needs to be changed. It’s now `https://downloads.services.belgium.be/eid/beidbuild-#{version}.dmg`

Comment (vitorgalvao) [inline Casks/eid-be.rb]: `pkg` is now also `eID-Quickinstaller-signed.pkg`.

Which means `uninstall` will also have to be rechecked for accuracy.

**Comment by fanquake** [comment]
Haven't been able to check the uninstall behaviour yet.

**Comment by franklouwers** [comment]
There are three packages:

- beid "middleware" (the driver + PKCS lib)
- beid "viewer": a java desktop GUI app that allows you to get information from a card, print out the contents, change the PIN etc
- beid "QuickInstaller", which wraps those two packages in one installer and adds a GUI installer. 

The old package only packaged the middleware. This Pull Request seems to package the QuickInstaller. My personal opinion is that there should be 2 casks: one for the middleware, another one for the GUI java app.

**Comment by vitorgalvao** [comment]
@fanquake How’s progress on this?

**Comment by thibmaek** [comment]
I wanted to give this a go, have a free day tomorrow and might get around to doing it.
Will create 2 seperate casks (`beid-viewer` & `beid-middleware`) and submit a PR.

**Comment by vitorgalvao** [comment]
@thibmaek I ask you to keep the `eid-be` nomenclature. You may add the `viewer` and `middleware` as suffixes.

A couple of questions, though:
+ Would it not be possible to just have the quickinstaller, since that seems to encapsulate the other two?
+ Does the viewer require the middleware to function?

**Comment by thibmaek** [comment]
@vitorgalvao oh yeah sure I was confused with how they named their packages instead of these cask names. While the quickinstaller suffices for most people, I and some others I know require only the middleware. The middleware is the only essential part in making belgian eid applications work, the viewer is just an extra to display information already on your card in digital form.

The viewer requires the middleware to function, the middleware packs the drivers etc.

**Comment by vitorgalvao** [comment]
> The viewer requires the middleware to function

In that case, be sure to add a [`depends_on cask:`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/depends_on.md#depends_on-cask) to the `viewer`.

**Comment by vitorgalvao** [comment]
Closing in favour of https://github.com/caskroom/homebrew-eid/pull/16.

**Comment by fanquake** [comment]
@thibmaek Thanks for cleaning this up 👍

**Pull request closed without merging**